AmazonChimeSDK-Bitcode.podspec

```diff
@@ -1,15 +1,15 @@
 Pod::Spec.new do |s|
   s.name             = 'AmazonChimeSDK-Bitcode'
-  s.version          = '0.19.0'
+  s.version          = '0.19.1'
   s.summary          = 'Amazon Chime SDK for iOS with Bitcode support.'
   s.description      = 'An iOS client library for integrating multi-party communications powered by the Amazon Chime service.'
   s.homepage         = 'https://github.com/aws/amazon-chime-sdk-ios'
   s.license          = 'Apache License, Version 2.0'
   s.author           = { 'Amazon Web Services' => 'amazonwebservices' }
-  s.source           = { :http => "https://amazon-chime-sdk-ios.s3.amazonaws.com/sdk/0.19.0/AmazonChimeSDK-0.19.0.tar.gz"}
+  s.source           = { :http => "https://amazon-chime-sdk-ios.s3.amazonaws.com/sdk/0.19.1/AmazonChimeSDK-0.19.1.tar.gz"}
   s.ios.deployment_target = '10.0'
   s.vendored_frameworks = "AmazonChimeSDK.xcframework"
   s.swift_version    = '5.0'
-  s.dependency 'AmazonChimeSDKMedia-Bitcode', '~> 0.15.0'
+  s.dependency 'AmazonChimeSDKMedia-Bitcode', '~> 0.15.1'
```

Comment:
do we need to bump up even before release?

Reply:
trying to put all version bump in one place. we are exposing this in preflight anyway.